.expeditor/Version_inspector/README.md

```diff
@@ -0,0 +1,11 @@
+#Version Inspector is a python based utility software that can be used to find the latest version information for the omnibus-software packages .
```

Comment:
single # is for largest headers, you may preview the readme [here](https://github.com/chef/omnibus-software/blob/test_version_update/.expeditor/Version_inspector/README.md) 
i believe you intend to have smaller text out there?